org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/panel/WDocActionPanel.java

```diff
@@ -379,7 +379,7 @@ public void onOk(final Callback<Boolean> callback) {
 				String docAction = lstDocAction.getSelectedItem().getLabel();
 				MessageFormat mf = new MessageFormat(Msg.getMsg(Env.getAD_Language(Env.getCtx()), "ConfirmOnDocAction"));
 				Object[] arguments = new Object[]{docAction};
-				FDialog.ask(0, this, mf.format(arguments), new Callback<Boolean>() {
+				FDialog.ask(0, this, "", mf.format(arguments), new Callback<Boolean>() {
```

Comment:
not the right fix, should replace the first argument, 0 with gridTab.getWindowNo() instead.

Reply:
done